lib/cart/cubits/cart_cubit.dart

```diff
@@ -1,25 +1,46 @@
+import 'dart:async';
+
 import 'package:bloc/bloc.dart';
-import 'package:flex_storefront/cart/apis/cart_api.dart';
+import 'package:flex_storefront/cart/bloc/global_cart_bloc.dart';
 import 'package:flex_storefront/cart/cubits/cart_state.dart';
 import 'package:flex_storefront/shared/bloc_helper.dart';
 
 class CartCubit extends Cubit<CartState> {
```

Comment:
In the same way we have the `CartIconCubit` around the `CartIcon`, maybe this cubit around the `CartPage` could be a `CartPageCubit`

That being said, I think we should aim for multiple cubits with a finer scope than the whole page. I imagine a cubit for the product list, a cubit for the total section, etc.